BookingsApi/BookingsApi.Infrastructure.Services/Dtos/UserDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Enumerations;
+
+namespace BookingsApi.Infrastructure.Services.Dtos
+{
+    public class UserDto
+    {
+        public string UserId { get; set; }
+        public string Username { get; set; }
+        public string UserRole { get; set; }
```

Comment:
Could maybe name this UserRoleName for clarity?

Reply:
I agree. Name changed